wdl-parsing/README.md

```diff
@@ -0,0 +1,120 @@
+# wdl-parsing
+
+This project contains source code and supporting files for a serverless application that you can deploy with the SAM CLI. It includes the following files and folders.
```

Comment:
Should give credit to wherever this came from
https://github.com/AWS-Bajio/promo-bot-mx-lambda
?

Reply:
That's not where it came from.  This README.md is generated from the SAM CLI and left as-is for the most part

Reply:
Ok, add a comment saying that this was generated with `sam init`(?)

Reply:
added markdown comment